example/provider-extensions/garden/controller-registrations/create-controller-registrations.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses~LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+usage() {
+  echo "Usage:"
+  echo "> create-controller-registrations.sh [ -h | <garden-kubeconfig> ]"
+  echo
+  echo ">> For example: create-controller-registrations.sh ~/.kube/garden-kubeconfig.yaml"
+
+  exit 0
+}
+
+if [ "$1" == "-h" ] || [ "$#" -ne 1 ]; then
+  usage
+fi
+
+garden_kubeconfig=$1
+
+echo "Creating controller-registrations"
+kubectl --kubeconfig $garden_kubeconfig --server-side=true apply -f https://raw.githubusercontent.com/gardener/gardener-extension-provider-gcp/v1.25.1/example/controller-registration.yaml
```

Comment:
I think it should be possible to pass additional values and to overwrite the version. We should then probably always use the latest release.

Reply:
We should probably not get into the business of maintaining their versions here in g/g. Instead, it should be the responsibility of the user to create these resources (with whatever custom configuration they need). Hence, I vote for dropping this file.